bats/tests/helpers/defaults.bash

```diff
@@ -74,6 +74,33 @@ if using_networking_tunnel && ! is_windows; then
     fatal "RD_USE_NETWORKING_TUNNEL only works on Windows"
 fi
 
+########################################################################
+: "${RD_MOUNT_TYPE:=reverse-sshfs}"
+
+validate_enum RD_MOUNT_TYPE reverse-sshfs 9p virtiofs
+
```

Comment:
Please, add platform check is_unix and fatal error for unix specific settings. Or do we just populate this settings on windows and just ignore them?